code/controllers/subsystem/SSblackbox.dm

```diff
@@ -327,7 +327,8 @@ SUBSYSTEM_DEF(blackbox)
 			"oxyloss" = L.getOxyLoss(),
 			"coord" = "[L.x], [L.y], [L.z]",
 			"server_id" = GLOB.configuration.system.instance_id,
-			"rid" = GLOB.round_id
+			"rid" = GLOB.round_id,
+			"last_words" = L.say_log.len > 0 ? L.say_log[L.say_log.len] : null
```

Comment:
```suggestion
			"last_words" = length(L.say_log) > 0 ? L.say_log[L.say_log.len] : null
```
First one needs to be `length()` which includes a null check for safety, but if it passes that the list will exist, and I *think* reading `.len` is faster than passing it through `length()` due to it being a VarGet not a proc jump.

Reply:
Kapu and some other guy said otherwise
https://discord.com/channels/484170914754330625/484280891158560778/1132773931544236202